Backend/DbSeed/DbSeed.cs

```diff
@@ -36,5 +41,28 @@ public static void CreateTestingUser(IServiceProvider serviceProvider)
                 usersCollection.InsertOne(user);
             }
         }
+
+        public static void CreateTestingCourses(IServiceProvider serviceProvider)
+        {
+            var dbService = serviceProvider.GetRequiredService<IMongoDatabase>();
+            var courseCollection = dbService.GetCollection<Course>(nameof(Course));
+            
+            var path = serviceProvider.GetRequiredService<IOptions<PathSettings>>();
+            var json = File.ReadAllText(path.Value.CoursesJsonPath);
```

Comment:
File.ReadAllText is throwing a lot of exceptions... Check if file exists before calling ReadAllText().

Reply:
this function is in try catch block in startup - it is useless handling it on two places